### PR TITLE
Update pipeline and add initial helm chart files

### PR DESCRIPTION
**Pull request by PhysicsJon**

### CODE REVIEW

File: domain-template.yaml [hunk 1]
```diff
@@ -4,8 +4,5 @@ services:
     extra_hosts:
       - "DOMAIN:IP"
   chrome:
-    extra_hosts:
-      - "DOMAIN:IP"
-  firefox:
     extra_hosts:
       - "DOMAIN:IP"
```
Comment (bjss-chris) [inline domain-template.yaml:8]: Just missing the trailing newline for git.

File: charts/of-ac-tests/templates/job.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          {{- if .Values.env.secrets -}}
+          {{- range .Values.env.secrets }}
+            - name: {{ .name }}
+              valueFrom:
+                secretKeyRef:            
+                  name: {{ .valueFrom.secretKeyRef.name }}
+                  key: {{ .valueFrom.secretKeyRef.key }}            
+          {{- end -}}
+          {{- end }}
+      restartPolicy: OnFailure
```
Comment (bjss-chris) [inline charts/of-ac-tests/templates/job.yaml:101]: Just missing a trailing newline for git.

File: azure-pipelines.yml [hunk 3]
```diff
@@ -1,103 +1,113 @@
...
+        -u "gpitfuturesdevacr" \
+        -p $(gpitfuturesdevacr-pass) \
+        $chartPackage
+  failOnStderr: false
+  condition: and(always(), or(eq(variables['Build.SourceBranch'], 'refs/heads/development'),eq(variables['Build.SourceBranch'], 'refs/heads/master')))
+  name: helmPush
+  displayName: 'Push helm chart to acr'
+
+
+
```
Comment (bjss-chris) [inline azure-pipelines.yml:113]: A few extra trailing lines here that can be trimmed?

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,7 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine3.11
+ENV MSBUILDSINGLELOADCONTEXT=1
+WORKDIR /app
+COPY ./src ./entrypoint.sh ./
+RUN apk add --no-cache bash && apk add curl
```
Comment (bjss-chris) [inline Dockerfile:5]: Might it be worth doing an `apk update` before the adds?

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+RUN dotnet publish -c Release -o out
+ENTRYPOINT ["/bin/bash", "entrypoint.sh"]
```
Comment (bjss-chris) [inline Dockerfile:7]: Just missing a trailing newline for git.

**Review by bjss-chris — COMMENTED** [review]
A couple of comments but the code and Docker changes look good. Recommend @gavinwakeley1 might be better to review the helm changes though.

**Review by gavinwakeley1 — APPROVED** [review]
Looks good from helm side

File: charts/of-ac-tests/templates/job.yaml [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
```
Comment (bjss-chris) [inline charts/of-ac-tests/templates/job.yaml:102]: Looks like git/GitHub still isn't happy for some reason.

File: domain-template.yaml [hunk 7]
```diff
@@ -6,6 +6,4 @@ services:
   chrome:
     extra_hosts:
       - "DOMAIN:IP"
-  firefox:
-    extra_hosts:
-      - "DOMAIN:IP"
\ No newline at end of file
+
```
Comment (bjss-chris) [inline domain-template.yaml]: Looks like git/GitHub still isn't happy for some reason (as above).

**Review by bjss-chris — APPROVED** [review]
👍

**Pull request merged**